servers/rendering/renderer_rd/shaders/scene_forward_clustered.glsl

```diff
@@ -969,6 +969,13 @@ void fragment_shader(in SceneData scene_data) {
 #endif //not render depth
 	/////////////////////// LIGHTING //////////////////////////////
 
+// FIXME: Run this block only if the shader writes into the VERTEX variable
+#ifdef USE_MULTIVIEW
+	view = -normalize(vertex_interp - scene_data.eye_offset[ViewIndex].xyz);
+#else
+	view = -normalize(vertex_interp);
```

Comment:
Shouldn't this be reading from ``vertex`` instead of ``vertex_interp``? As ``vertex`` is the variable that changes if ``VERTEX`` is used.

Reply:
Nice catch, fixing it. Classic copy-paste then forget to modify it mistake... 🤦‍♂️

Reply:
Fixed it